pyqtgraph/parametertree/parameterTypes/checklist.py

```diff
@@ -0,0 +1,169 @@
+from .basetypes import GroupParameterItem, GroupParameter, WidgetParameterItem
+from .list import ListParameter
+from .. import ParameterItem
+from ...Qt import QtWidgets
+from ... import functions as fn
+
+class ChecklistParameterItem(GroupParameterItem):
+    """
+    Wraps a :class:`GroupParameterItem` to manage ``bool`` parameter children. Also provides convenience buttons to
+    select or clear all values at once. Note these conveniences are disabled when ``exclusive`` is *True*.
+    """
+    def __init__(self, param, depth):
+        self.btnGrp = QtWidgets.QButtonGroup()
+        self.btnGrp.setExclusive(False)
+        self._constructMetaBtns()
+
+        super().__init__(param, depth)
+
+    def _constructMetaBtns(self):
+        self.metaBtnWidget = QtWidgets.QWidget()
```

Comment:
@j9ac9k Notice `_constructMetaBtns` doesn't assign a parent to the widget buttons, is this why the palette change isn't detected? I can't reproduce either way on my laptop, but I'm curious if the button text is correct for you if you change this to `QWidget(self)`.

I figured line 42 (`tw.setItemWidget(self, 1, self.metaBtnWidget)`) would properly assign the parent, but maybe I'm wrong

Reply:
changing

```python
        self.metaBtnWidget = QtWidgets.QWidget()
```

to

```python
        self.metaBtnWidget = QtWidgets.QWidget(self)
```

Results in a type-error.